File: test/SampleTask.js
var readline = require('readline');
var answer = require('./answer.js');

var rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
  terminal: false
});

rl.on('line', function (line) {
  const arg1 = line.charCodeAt(0);
  const arg2 = line.charCodeAt(1);
  console.log(answer.answer(arg1, arg2));
  process.exit();
});

File: test/SampleTask.py
# import user code
from answer import answer
import sys

def test(data):
  arg1 = ord(data[0])
  arg2 = ord(data[1])
  print(answer(arg1, arg2))

if __name__ == "__main__":
  a = sys.stdin.readline()
  test(a)
